### PR TITLE
[metricsadvisor] live tests setup

### PR DESCRIPTION
**Pull request by kristapratico**
Once checked into master, will run pipeline generator and make any adjustments necessary in another PR

### CODE REVIEW

File: sdk/metricsadvisor/tests.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    parameters:
+      BuildTargetingString: azure-ai-metricsadvisor
+      ServiceDirectory: metricsadvisor
+      EnvVars:
+        AZURE_SUBSCRIPTION_ID: $(provisioner-subscription)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_METRICS_ADVISOR_ENDPOINT: $(metricsadvisor-test-service-endpoint)
+        AZURE_METRICS_ADVISOR_SUBSCRIPTION_KEY: $(metricsadvisor-test-subscription-key)
```
Comment (xiangyan99) [inline sdk/metricsadvisor/tests.yml:14]: Do we share these variables with other language?

File: sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py [hunk 2]
```diff
@@ -428,13 +428,13 @@ def __init__(self, method_name):
...
-            self.anomaly_detection_configuration_id = self.get_settings_value("ANOMALY_DETECTION_CONFIGURATION_ID")
-            self.anomaly_alert_configuration_id = self.get_settings_value("ANOMALY_ALERT_CONFIGURATION_ID")
-            self.metric_id = self.get_settings_value("METRIC_ID")
-            self.incident_id = self.get_settings_value("INCIDENT_ID")
-            self.dimension_name = self.get_settings_value("DIMENSION_NAME")
-            self.feedback_id = self.get_settings_value("FEEDBACK_ID")
-            self.alert_id = self.get_settings_value("ALERT_ID")
+            self.anomaly_detection_configuration_id = "c0f2539f-b804-4ab9-a70f-0da0c89c76d8"
+            self.anomaly_alert_configuration_id = "ff3014a0-bbbb-41ec-a637-677e77b81299"
+            self.metric_id = "3d48ed3e-6e6e-4391-b78f-b00dfee1e6f5"
```
Comment (xiangyan99) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py:433]: Why we hard code the id here?

File: sdk/metricsadvisor/tests.yml (hunk 1 above)
Comment (kristapratico) [reply sdk/metricsadvisor/tests.yml:14]: Yes, all environment variables are shared across languages except API key

File: sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py (hunk 2 above)
Comment (kristapratico) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py:433]: So we don't have to add them to the test secrets keyvault.

Comment (xiangyan99) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py:433]: But if we modify the secrets that we saved in keyvault, we have to update them as well, right?

I mean these ids are coupled with other secrets.

Comment (kristapratico) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py:433]: yeah that's a good point. We were trying to minimize the number of things we were adding to the keyvault, but I think it probably makes more sense to allow ease of update here. I'll add them back as env vars.

**Review by xiangyan99 — APPROVED** [review]

**Pull request merged**